benchmarks/benchmarks/optimize.py

```diff
@@ -209,7 +209,8 @@ def bench_run(self, x0, methods=None, **minimizer_kwargs):
         if methods is None:
             methods = ["COBYLA", 'Powell',
                        'L-BFGS-B', 'BFGS', 'CG', 'TNC', 'SLSQP',
-                       "Newton-CG", 'dogleg', 'trust-ncg', 'trust-region-exact']
+                       "Newton-CG", 'dogleg', 'trust-ncg', 'trust-trlib',
```

Comment:
And what about ``trust-krylov``? It gives the right idea and the fact that it doesn't tells the exact difference between it and ``trust-ncg`` doesn't seems a big problem to me. (``Newton-CG``and ``trust-ncg``, for instance, have names that doesn't specify the algorithm entirely either...)